jsons/Policies.json

```diff
@@ -403,7 +403,7 @@
 		"policies": [
 			{
 				"name": "Silk Road",
-				"uniques": ["+[25]% yield from every [Land Trade Route (Food)]","+[25]% yield from every [Land Trade Route (Production)]","+[25]% yield from every [Sea Trade Route (Food)]","+[25]% yield from every [Sea Trade Route (Production)]"],
+				"uniques": ["[+25]% yield from every [Land Trade Route (Food)]","[+25]% yield from every [Land Trade Route (Production)]","[+25]% yield from every [Sea Trade Route (Food)]","[+25]% yield from every [Sea Trade Route (Production)]","[+25]% yield from every [Sea Trade Route (Gold)]","[+25]% yield from every [Land Trade Route (Gold)]"]
```

Comment:
I think Yield might be capitalized here, not sure

Reply:
resolved in https://github.com/ravignir/RekMOD/pull/43/commits/d98b1105a0e0d4f0661c3954f3b1811a2899ab7c